libs/blocks/aside/aside.css

```
@@ -104,6 +130,11 @@
   margin-bottom: var(--spacing-s);
 }
 
+.aside .foreground.container .text .supplemental-text {
+  padding-top: var(--spacing-s);
+  margin-bottom: 0;
+}
+
 .aside .foreground.container .text .action-area {
   margin-bottom: 0;
```

```suggestion
  margin-bottom: 0;
  display: flex;
  flex-wrap: wrap;
  align-items: flex-start;
  gap: var(--spacing-xs);
```
In small viewports the CTA buttons will stack on top of each other w/ out spacing. This will give us better defaults for spacing and remove some unneeded code below.

---

@ryanmparrish This fix was made as a part of the PR https://github.com/adobecom/milo/pull/1016 which is merged and the issue is not present now.